### ISSUE TITLE
Error in sandbox mode: attempt to index field 'character' (a nil value)

### ISSUE THREAD

**Issue by keryja** [Suprcheese/Satellite-Uplink-Station#2]
If you have Satellite Uplink Station loaded while opening the sandbox scenario it'll throw an error as soon inventory changes, as there's no player character.

```
Error while running event on_player_quickbar_inventory_changed (ID 33)
__Satellite Uplink Station__/control.lua:69: attempt to index field 'character' (a nil value)
```
# Reproduction Steps
1. Play -> Custom Scenario -> Sandbox -> Generate
2. Click yes on all tech and items
3. Error appears as soon as items get added

**Comment by Suprcheese** [comment]
Ah, yeah the uplink station was not designed with the sandbox scenario in mind. I'll probably just disable it if no player.character is detected.

**Comment by Suprcheese** [comment]
Resolved in version 1.0.3.
